### PR TITLE
Test `CREATE TABLE AS SELECT` in Ignite type mapping 

### LINKED ISSUE #16214: Support Trino CTAS data setup process for Ignite connector
Added `trinoCreateAsSelect` DataSetup for Ignite sql type mapping test.
Follow up for https://github.com/trinodb/trino/pull/8323

### PR DESCRIPTION
**Pull request by sudip-unb**
## Description

Fixes #16214

## Release notes

(x) This is not user-visible or docs only and no release notes are required.

### CODE REVIEW

**Review by ebyhr — APPROVED** [review]
Please submit CLA if you haven't sent it.

**Comment by ebyhr** [comment]
Please take a look at CI failure.

**Comment by sudip-unb** [comment]
Working on it.

**Comment by ebyhr** [comment]
@martint Could you confirm CLA from @sudip-unb ?

File: plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteTypeMapping.java [hunk 1]
```diff
@@ -400,8 +426,23 @@ private DataSetup trinoCreateAndInsert(Session session, String tableNamePrefix)
         return new CreateAndInsertDataSetup(new TrinoSqlExecutor(getQueryRunner(), session), tableNamePrefix);
     }
 
+    private DataSetup trinoCreateAsSelect(String tableNamePrefix)
```
Comment (ebyhr) [inline plugin/trino-ignite/src/test/java/io/trino/plugin/ignite/TestIgniteTypeMapping.java:429]: `testDate` test method should use this method.

**Review by ebyhr — APPROVED** [review]

**Pull request merged**